install/kubernetes/helm/istio/charts/mixer/templates/deployment.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "mixer.fullname" . }}
+  labels:
+    app: {{ template "mixer.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  annotations:
+    sidecar.istio.io/inject: "false"
+    checksum/config-volume: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "mixer.name" . }}
+        release: {{ .Release.Name }}
+        istio: mixer
+    spec:
+      serviceAccountName: {{ template "mixer.serviceAccountName" . }}
+      containers:
+        - name: statsd-to-prometheus
+          image: {{ .Values.prometheus.repository }}:{{ .Values.prometheus.tag}}
+          imagePullPolicy: {{ .Values.prometheus.imagePullPolicy }}
+          ports:
+          - containerPort: 9102
```

Comment:
If your goal to run multiple instances, then ports need to become variables

Reply:
you can run multiple instances that expose same port, or you mean to better differentiate between promtheus targets?